### DISCUSSION TITLE
move to another space and launch

### DISCUSSION THREAD

**Discussion by ed-tobias** [ttscoff/bunch-docs#338] [Q&A]
I want to launch a few apps then move to another space (using control-right arrow) and launch a few more apps.  I can't seem to get bunch to move to another space. I've tried focus on Finder followed by - {^right} with no luck. Is this not possible to do?

**Comment by ttscoff** [comment]
You don't want to send the keystrokes to finder, they should be global. Just leave the hyphen off. That said, there have been very mixed results with this kind of thing.

- Brett

**Reply by ed-tobias** [reply]
I could not get it to work, however using btt to navigate spaces and calling it within bunches works every time!
